sharing/process/index.md

```diff
@@ -0,0 +1,53 @@
+---
+tags:
+  - sharing
+author:
+  display_name: AGRC Office
+  email: agrc@utah.gov
+date: 2017-01-26 23:40:06
+title: Submitting Data to the SGID
+categories: []
+---
+
+Thank you for wanting to share your data with the SGID! We've put together a few processes to make sure your datasets are added to the right place. We use [issues](https://guides.github.com/features/issues/) in our [Porter](https://github.com/agrc/porter/issues) repository on GitHub to track where a dataset is at in the process and keep an open and transparent record of everything that happened.
+
+If you're not sure which part of the SGID you should share your data with, check out our sharing [introduction]({% link sharing/index.md %}#join-the-party) for an overview.
+
+If you've still got questions after reading this, {% include contact.html subject="SGID Data Additions" contact=site.data.contacts.sgid text='reach out to ' hide-punctuation=true %}, or anyone else on the [AGRC team]({% link about/contact/index.html %}).
+
+## Contributing Data to an Existing Dataset
+
+If you have data you'd like to add to an existing dataset, please check the data page for the contact info of the AGRC staffer responsible. If none are listed, please reach out via our main email. We are happy to work with authoritative stewards and other qualified individuals to improve our data
+
+## Submitting New Datasets
+
+No matter where your datasets fit best—SGID Index, Open Data, or Open SGID—the process begins with the same first two steps: first, making sure your datasets meet the relevant [qualifications]({% link about/policy/sgid/index.md %}) and then opening an issue in [Porter](https://github.com/agrc/porter/issues) using the relevant template.
```

Comment:
**thought**: we should consider offering help to fill this out for first timers as it is probably a bit of a scary task. After doing it once, it will probably be more approachable without hand holding? Maybe a quick movie could be used to describe the process?